src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -330,10 +332,37 @@ private void addConfig(Map<String, ?> parsedProps, ComposableConfig config) {
     }
   }
 
+  /**
+   * Returns the url property. Preference is given to property <code>store.url</code> over
+   * <code>hdfs.url</code> because <code>hdfs.url</code> is deprecated.
+   *
+   * @param props - Map of properties for the connector
+   * @return String - url for HDFS
+   */
+  private String extractUrl(Map<String, String> props) {
+    String storageUrl = props.get(StorageCommonConfig.STORE_URL_CONFIG);
+    if (storageUrl != null && !storageUrl.equals(StorageCommonConfig.STORE_URL_DEFAULT)) {
```

Comment:
But also taking a step back, not sure what are we aiming for here.

For example, not sure why we don't use `getString`
There's no very elegant way to write this. Here's a suggestion that minimizes lines (instead of calls): 
```
    if (StringUtils.isNotBlank(getString(STORE_URL_CONFIG))) {
      return getString(STORE_URL_CONFIG);
    }
    if (StringUtils.isNotBlank(getString(HDFS_URL_CONFIG))) {
      return getString(HDFS_URL_CONFIG);
    }
    throw new ConfigException(
        String.format("Configuration %s cannot be empty.", StorageCommonConfig.STORE_URL_CONFIG
    );
```